integration_go.mod

```diff
@@ -0,0 +1,91 @@
+module gopkg.in/DataDog/dd-trace-go.v1 // This file is manually maintained and updated
+
+go 1.22.0
```

Comment:
Could we run a `go mod tidy` in this directory? There's some contribs that we've removed support for in v2 that are still in here, so I think a tidy might help take care of those.

Or, if this is manually updated, let's just take a look at the `contrib` folder and see what's still there and what's not. If you'd like to connect on that, feel free to find me!

Reply:
done!